src/WebUI/Types.d.ts

```diff
@@ -38,6 +38,7 @@ export interface IServiceItem {
     port: string;
     creationTimestamp: Date;
     uid: string;
+    pipeline: string;

```

Comment:
why this is mandatory?

Reply:
you could always pass-in an empty string if there is no pipeline information (which is what our Utils.getPipelineText method does). It's easier this way rather than having an optional member and then checking for validity while using/rendering it.